packages/SwingSet/test/test-vat-env.js

```diff
@@ -35,7 +34,6 @@ test('kind makers are in the test environment', t => {
 
 test('store makers are in the test environment', t => {
   // TODO: configure eslint to know that VatData is a global
```

Comment:
A PR to just configure eslint to know that VatData is a global would be good. It should remove this TODO.

Reply:
ref https://github.com/Agoric/agoric-sdk/wiki/Add-global-variables-to-ESLint-and-JSDoc-annotations-validated-by-Typescript